src/app/project/ontology/property-form/property-form.component.spec.ts

```diff
@@ -50,29 +164,107 @@ xdescribe('PropertyFormComponent', () => {
                 {
                     provide: DspApiConnectionToken,
                     useValue: new KnoraApiConnection(TestConfig.ApiConfig)
```

Comment:
We should probably mock the DSP-API connection like we do in our other tests

Reply:
I see. I think this is the "old" way. I found the same setup in 23 other files which have to be refactored in another branch. 
Resolved this one in baf0a41